pkg/schema/di.go

```diff
@@ -15,6 +17,33 @@ var cr *CoreRegistry
 // atomic guard on registry
 var regguard int32
 
+var schm *runtime.Scheme = runtime.NewScheme()
+
+var (
+	groupName    = "grafana.ap.group"
```

Comment:
`ap.group`? it's fine if this is just WIP stuff, i'm just curious where it's coming from

Reply:
TBH I don't know: I think I was planning to have `grafana.core.group` there but I'm good with whatever we decide.